try:
    s = round('32.156666', 2)
except ValueError as s:
    print(s)
